hw9/modules/album.ts

```diff
@@ -0,0 +1,24 @@
+import type {MainClassesTypes} from "./_typesGuardMainClass";
+
+export class Album{
+    constructor(
+        public name: string,
+        public dateRelease: MainClassesTypes, 
+        public tracks?: MainClassesTypes, 
+        public artist?: MainClassesTypes){
+            //TYPE GUADRD
```

Comment:
it's not a type guard

Reply:
type guard is a signle function.
example takes from [link](https://basarat.gitbook.io/typescript/type-system/typeguard#user-defined-type-guards)

``` ts
/**
 * User Defined Type Guard!
 */
function isFoo(arg: any): arg is Foo {
    return arg.foo !== undefined;
}
```